From e6abfa9691d33877c8cf8bbbc0e96536f191b3c9 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Fri, 30 Jan 2015 16:33:16 +0800
Subject: [PATCH] ENGR00309843 Boot: No logo displayed during booting up

No splash screen in 3.10 kernel, have to enable below for android log
Add CONFIG_LOG and FRAMEBUFFER_CONSOLE
The booting up log have below sequence:
   uBoot logo --> Kernel Logo --> "ANDROID" in framebuffer console
   --> Android's bootanimation

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arch/arm/configs/imx_v7_android_defconfig | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/imx_v7_android_defconfig b/arch/arm/configs/imx_v7_android_defconfig
index 51d1ab5..984791c 100644
--- a/arch/arm/configs/imx_v7_android_defconfig
+++ b/arch/arm/configs/imx_v7_android_defconfig
@@ -294,7 +294,6 @@ CONFIG_INPUT_KEYCHORD=y
 CONFIG_INPUT_UINPUT=y
 CONFIG_INPUT_GPIO=y
 CONFIG_INPUT_ISL29023=y
-# CONFIG_VT is not set
 # CONFIG_LEGACY_PTYS is not set
 # CONFIG_DEVKMEM is not set
 CONFIG_SERIAL_IMX=y
@@ -381,6 +380,8 @@ CONFIG_FB_MXC_EINK_PANEL=y
 CONFIG_FB_MXS_SII902X=y
 CONFIG_FB_MXC_DCIC=y
 CONFIG_HANNSTAR_CABC=y
+CONFIG_FRAMEBUFFER_CONSOLE=y
+CONFIG_FRAMEBUFFER_CONSOLE_DETECT_PRIMARY=y
 CONFIG_LOGO=y
 CONFIG_SOUND=y
 CONFIG_SND=y
@@ -568,3 +569,6 @@ CONFIG_CRC_CCITT=y
 CONFIG_CRC_T10DIF=y
 CONFIG_CRC7=y
 CONFIG_LIBCRC32C=y
+CONFIG_FONTS=y
+CONFIG_FONT_8x8=y
+CONFIG_FONT_8x16=y
-- 
1.8.0

